test/fixtures/tasks/tasks.js

```diff
@@ -67,5 +67,17 @@ module.exports = () => {
 
       isNoteworthy: true,
     },
+    {
+      title: "check is task form is working",
```

Comment:
I think you wanted to write 
```suggestion
      title: "Check if task form is working",
```